constructor.sh

```diff
@@ -148,13 +164,15 @@ TMP_DIR=$(mktemp -d)
 BASE=$BASE_PARENT/elastic
 
 # Check that the user exists
-if ! grep -q "^${USER}:" /etc/passwd; then
-	adduser --disabled-login --system --home $BASE $USER
+if ! grep -q "^${ES_USER}:" /etc/passwd; then
+    proxy_log "adding user $ES_USER"
+	adduser --disabled-login --system --home $BASE $ES_USER
 fi
 
 # sometimes (I'm looking at you, Hetzner) we can find ourselves with a
 # bad IPv6 configuration. If so, disable it here.
 if [[ $DISABLE_IPV6 ]]; then
+    proxy_log "disabling ipv6"
```

Comment:
The indentation in this file could be fixed. I think a modeline at the bottom could help

http://vim.wikia.com/wiki/Modeline_magic

Reply:
I'm using Atom editor at the moment. Modelines only work for vim I think. Atom *should* fix indentation automatically, but it only does so consistently for new text.

Is there not a continuous-integration trick we can use in github? That would make it client-agnostic...

Reply:
Fixed in this instance, but we should look at something more automated